File: nvxapp.server.service/ClientServer-Service/Account/Models/FinancialAdvisorListModel.cs
using nvxapp.server.data.Entities.Public;
using nvxapp.server.service.ClientServer_Service.ModelsBase;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using static nvxapp.server.data.Entities.AspNetUsersDataUtil;

namespace nvxapp.server.service.ClientServer_Service.Account.Models
{
    public class FinancialAdvisorListInModel
    {
    }

    public class FinancialAdvisorListOutModel : ModelResult
    {
        public List<FinancialAdvisorListModel> FinancialAdvisorList { get; set; }

        public FinancialAdvisorListOutModel()
        {
            FinancialAdvisorList = new List<FinancialAdvisorListModel>();
        }
    }

    public class FinancialAdvisorListModel
    {
        public int IdFinancialAdvisor { get; set; }
        
        public required string IdAspNetUsers { get; set; }

        public string? Descrizione { get; set; }
        
    }

}

File: nvxapp.client/src/nvx/ClientServer-Service/Account/Models/financial-advisor-list-model.ts
import { ModelResult } from "../../ModelsBase/model-result";
import { RolesModel } from "./user-roles-model";


export class FinancialAdvisorListInModel {
  
}

export class FinancialAdvisorListOutModel extends ModelResult {

  public financialAdvisorList: FinancialAdvisorListModel[] ;

}

export class FinancialAdvisorListModel  {

  public idFinancialAdvisor: number;

  public descrizione: string | null;

  public idAspNetUsers: string ;

}

